src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -95,16 +105,19 @@
          </widget>
         </item>
         <item>
-         <widget class="QCheckBox" name="apiCheckBox">
+         <widget class="QCheckBox" name="gpuExecutionsCheckBox">
           <property name="text">
-           <string>Enable Orbit Api in target</string>
+           <string>Trace GPU command buffer executions</string>
```

Comment:
I'm not sure this is the best way of describing it, let me know if you have a better one in mind (similarly for the corresponding "trace gpu executions" functions and variables).

Reply:
Great, I went with "Trace GPU queue submissions" since we have both "queue" and "submissions" in the UI (and with "(trace) gpu submissions" for variable/function names).

Reply:
Yeah, I think that's the best choice as it's clear what part in the UI it refers to.